lib/src/ably_implementation.dart

```diff
@@ -1,19 +1,20 @@
 import 'dart:async';
 
-import 'package:ably_flutter_plugin/src/impl/message.dart';
 import 'package:ably_flutter_plugin/src/interface.dart';
+import 'package:flutter/material.dart';
```

Comment:
Is this an unused import?

Reply:
Yes, thanks. Removed da5de0f49b0b5f6f6aa7f3ea269a088e0c33be1a

Reply:
Thanks.

I think I spotted that when I pulled this in to my IDE.

Are we not running a linter? I thought `pedantic` was installed in this project but maybe I forget how all that hooks in. I would prefer that we ran as much lint / analysis as we can on every build if possible.

Reply:
Linter is not enabled in this I see. Should it be enabled via actions?

Reply:
No, don't worry for now. I'll sort this out later, outside of the scope of this PR.